### DISCUSSION TITLE
Error while using GetCACaps operation in SSECP

### DISCUSSION THREAD

**Discussion by 0xst3m** [Keyfactor/ejbca-ce#502] [Q&A]
It seems that the "GetCACaps" operation is executing first and giving an error when the "getca" operation is called. Is there anyway to fix this error?

`sscep getca -i "ManagementCA" -u http://localhost:80/ejbca/publicweb/apply/scep/testing/pkiclient.exe -c ca.crt -d                                                
sscep: starting sscep, version 0.10.0
sscep: new transaction
sscep: transaction id: SSCEP transactionId
sscep: hostname: localhost
sscep: directory: ejbca/publicweb/apply/scep/testing/pkiclient.exe
sscep: port: 80
sscep: SCEP_OPERATION_GETCAPS
sscep: scep request:
GET /ejbca/publicweb/apply/scep/testing/pkiclient.exe?operation=GetCACaps HTTP/1.1
Host: localhost
Connection: close

sscep: connecting to localhost:80
sscep: server response status code: 404, MIME header: text/html
<html><head><title>Error</title></head><body>The SCEP alias testing is in CA mode, the message parameter in the GET request is empty, and no default CA has been defined for SCEP. Either switch to RA mode, provide the name of the CA in the message, or specify the default CA using the scep.defaultca property.</body></html>
sscep: SCEP_OPERATION_GETCA
sscep: scep request:
GET /ejbca/publicweb/apply/scep/testing/pkiclient.exe?operation=GetCACert&message=ManagementCA HTTP/1.1
Host: localhost
Connection: close

sscep: connecting to localhost:80
sscep: server response status code: 200, MIME header: application/x-x509-ca-cert
sscep: valid response from server
sscep: MD5 fingerprint: 62:29:2D:84:6B:04:D8:16:4F:B8:F3:9B:F6:0C:E5:04
sscep: CA certificate written as ca.crt`

**Comment by mike-agrenius-kushner** [comment]
Closing, duplicate of #503 . Please don't x-post.
